app/src/main/kotlin/com/wire/android/ui/userprofile/other/OtherUserProfileScreen.kt

```diff
@@ -0,0 +1,194 @@
+                            userName = userName,
+                            teamName = teamName,
+                            isEditable = false
+                        )
+                    }
+
+                    item { Spacer(Modifier.height(16.dp)) }
+
+                    item {
+                        UserDetailInformation(
+                            title = stringResource(R.string.email_label),
+                            value = state.email,
+                            onCopy = { otherUserProfileScreenState.copy(it) }
+                        )
+                    }
+
+                    item {
+                        UserDetailInformation(
+                            title = stringResource(R.string.phone_label),
+                            value = state.phone,
+                            onCopy = { otherUserProfileScreenState.copy(it) }
+                        )
+                    }
+
+                }
+                Divider()
+                Row(
+                    horizontalArrangement = Arrangement.Center,
+                    verticalAlignment = Alignment.CenterVertically,
+                    modifier = Modifier
+                        .height(dimensions().groupButtonHeight)
+                        .fillMaxWidth()
+                        .padding(all = dimensions().spacing16x)
+                ) {
+                    WirePrimaryButton(
+                        text = stringResource(R.string.label_open_conversation),
+                        onClick = {
+                            //TODO:redirect to conversation
+                        },
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun UserDetailInformation(title: String, value: String, onCopy: (String) -> Unit) {
+    RowItemTemplate(
+        title = {
+            Text(
+                style = MaterialTheme.wireTypography.subline01,
+                color = MaterialTheme.wireColorScheme.labelText,
+                text = title.uppercase()
+            )
+        },
+        subtitle = {
+            Text(
+                style = MaterialTheme.wireTypography.body01,
+                color = MaterialTheme.wireColorScheme.onBackground,
+                text = value
+            )
+        },
+        actions = { CopyButton(onCopyClicked = { onCopy("$value copied") }) },
+        onRowItemClicked = { },
+        onRowItemLongClicked = { }
+    )
+}
+
+@Composable
+fun OtherUserProfileTopBar(onNavigateBack: () -> Unit) {
+    WireCenterAlignedTopAppBar(
+        onNavigationPressed = onNavigateBack,
+        title = stringResource(id = R.string.user_profile_title),
+        elevation = 0.dp,
+        actions = {
+            MoreOptionIcon({ })
+        }
+    )
+}
+
+
+@Composable
+fun rememberOtherUserProfileScreenState(
+    snackBarHostState: SnackbarHostState = remember { SnackbarHostState() },
+    coroutineScope: CoroutineScope = rememberCoroutineScope()
```

Comment:
Do you think it's necessary that or we'll need to pass this `coroutineScope` here ? In case the usage won't consider passing it as an argument, I think we could inline it inside the body and get rid of the param